lib/src/editor/ui/editor_tabs.dart

```diff
@@ -91,35 +94,42 @@ class _CoreContentTabsState extends State<CoreContentTabs> {
             ),
           ),
           const TabsVerticalDivider(),
-          StyledTextButton(
-            onPressed: context.read<ZoomProvider>().zoomIn,
-            icon: FeatherIcons.zoomIn,
-          ),
-          const SizedBox(width: 8),
-          StyledTextButton(
-            onPressed: context.read<ZoomProvider>().zoomOut,
-            icon: FeatherIcons.zoomOut,
-          ),
-          const SizedBox(width: 8),
-          StyledTextButton(
-            onPressed: context.read<ZoomProvider>().resetZoom,
-            icon: FeatherIcons.refreshCcw,
-          ),
-          StyledTextButton(
-            onPressed: context.read<GridProvider>().toggleGrid,
-            icon: FeatherIcons.grid,
-          ),
-          const TabsVerticalDivider(),
-          const SizedBox(width: 8),
+          context.read<TabProvider>().tab == FlutterBookTab.canvas
+              ? Row(
+                  children: [
+                    StyledTextButton(
+                      onPressed: context.read<ZoomProvider>().zoomIn,
+                      icon: FeatherIcons.zoomIn,
+                    ),
+                    const SizedBox(width: 8),
+                    StyledTextButton(
+                      onPressed: context.read<ZoomProvider>().zoomOut,
+                      icon: FeatherIcons.zoomOut,
+                    ),
+                    const SizedBox(width: 8),
+                    StyledTextButton(
+                      onPressed: context.read<ZoomProvider>().resetZoom,
+                      icon: FeatherIcons.refreshCcw,
+                    ),
+                    StyledTextButton(
+                      onPressed: context.read<GridProvider>().toggleGrid,
+                      icon: FeatherIcons.grid,
+                    ),
+                    const TabsVerticalDivider(),
+                    const SizedBox(width: 8),
+                    StyledTextButton(
+                      onPressed:
+                          context.read<DevicePreviewProvider>().togglePreview,
+                      icon: FeatherIcons.smartphone,
+                    ),
+                    const SizedBox(width: 8),
```

Comment:
Would declare a variable for `SizedBox(width: 8)` since it is used multiple times in this file